module05/calculator.py

```diff
@@ -16,25 +16,31 @@ def multiply(x, y):
 def divide(x, y):
    return x / y
 
-print("Select operation.")
-print("1.Add")
-print("2.Subtract")
-print("3.Multiply")
-print("4.Divide")
 
-# Take input from the user 
-choice = int(input("Enter choice(1/2/3/4):"))
-
-num1 = int(input("Enter first number: "))
-num2 = int(input("Enter second number: "))
-
-if choice == 1:
-   print(num1,"+",num2,"=", add(num1,num2))
-elif choice == 2:
-   print(num1,"-",num2,"=", subtract(num1,num2))
-elif choice == 3:
-   print(num1,"*",num2,"=", multiply(num1,num2))
-elif choice == 4:
-   print(num1,"/",num2,"=", divide(num1,num2))
-else:
-   print("Invalid choice", choice)
+# if __name__ == '__main__':
```

Comment:
@sstoyanov7 , it is indeed in some way because [Module 05 - README](https://github.com/sstoyanov7/qa-automation-python-selenium-101/tree/master/module05) -  says: 

*Modify calculator.py to avoid executing the interactive commands when testing*